src/components/MealForm/MealForm.stories.js

```diff
@@ -0,0 +1,8 @@
+import MealForm from './MealForm'
```

Comment:
You could also add a `Input.stories.js` and `Input.spec.js` where you show this component and test it.